### PR TITLE
Merge master into next

### PR DESCRIPTION
**Pull request by susnux**
Merge master into next to prepare for a new alpha version.

### CODE REVIEW

File: src/components/NcActions/NcActions.vue [hunk 1]
```diff
@@ -716,8 +716,8 @@ export default {
 			</NcActions>
 		</p>
 
-		<h2>Tooltip</h2>
-		Has only text and not interactive elements
+		<h2>Toolip</h2>
```
Comment (Antreesy) [inline src/components/NcActions/NcActions.vue:719]: Is it on master as well?

**Review by Antreesy — APPROVED** [review]

File: src/components/NcActions/NcActions.vue (hunk 1 above)
Comment (susnux) [reply src/components/NcActions/NcActions.vue:719]: Did not change this - this is from the merge. So yes

Comment (susnux) [reply src/components/NcActions/NcActions.vue:719]: Ah noticed the typo :see_no_evil:
That should not be backported

Comment (susnux) [inline src/components/NcActions/NcActions.vue:719]:
```suggestion
		<h2>Tooltip</h2>
```

**Pull request closed without merging**